.bazelrc

```diff
@@ -1,3 +1,8 @@
+# Required for Bazel versions <7, where bzlmod is not enabled by default
```

Comment:
How about we also recommend using bazelisk on the README?

Reply:
Done.